functions/helpers.bats

```diff
@@ -2,30 +2,6 @@
 
 load helpers.bash
 
-testNonExistingHost() {
```

Comment:
why delete?

Reply:
It is not applicable to the openhabian base system and often fails resulting in a bad run.

Reply:
It's a unit test I've built for the tryUntil() function on @EliasGabrielsson 's request some time ago. I have not seen it fail to date. If it fails due to some in-Docker-weirdnesses you should at least replace it.

Reply:
No, it just occasionally fails when run in Travis and causes a full fail. Why do we need to have a test case for that when it is tested 7 times on each build of the travis run once first-boot.bash is run.

Reply:
Yes no need to separately test - unless we change anything about that function.
I also found that a clever way to test. Could you just inactivate it by changing the test name like unit-*  => inactive-*

Reply:
That is pretty ingenious, sure I have not problem with that.